website/docs/audit.md

```diff
@@ -13,7 +13,8 @@ There are three ways to gather audit results, depending on the level of detail n
 
 Prometheus metrics provide an aggregated look at the number of audit violations:
 
-* `gatekeeper_audit_last_run_time` provides the timestamp of the most recently completed audit run
+* `gatekeeper_audit_last_run_time` provides the timestamp of the most recently started audit run
+* `gatekeeper_audit_last_run_end_time` provides the timestamp of the most recently finished audit run
```

Comment:
```suggestion
* `gatekeeper_audit_last_run_end_time` provides the end time timestamp of the most recent audit run
```

Reply:
I've replaced `the most resent` with `the last completed` - please tell me if you'd still prefer other wording